backend/app/views.py

```diff
@@ -144,6 +151,7 @@ def get_picturebook_data(request):
     API endpoint for getting the picture book images given the story the user wrote.
     """
     story_content = request.query_params.get('content')
+    print('hmmm:', story_content)
```

Comment:
Looks like this is some debugging that was left in place.

Reply:
The debugging statement has been removed